Template/Tag/CustomHtmlTag.web.js

```diff
@@ -101,25 +101,28 @@
 
             var src = TagManager.dom.getElementAttribute(element, 'src');
             if (src) {
-                newScript.src = src;
+                newScript.setAttribute('src', src);
             } else {
-                newScript.text = element.text || element.textContent || element.innerHTML || '';
+                newScript.setAttribute('text', (element.text || element.textContent || element.innerHTML || ''));
             }
 
-            if (element.id) {
-                newScript.id = element.id;
+            if (element.hasAttribute('id')) {
+                newScript.setAttribute('id', element.getAttribute('id'));
             }
-            if (element.charset) {
-                newScript.charset = element.charset;
+            if (element.hasAttribute('charset')) {
+                newScript.setAttribute('charset', element.getAttribute('charset'));
             }
-            if (element.defer) {
-                newScript.defer = element.defer;
+            if (element.hasAttribute('defer')) {
+                newScript.setAttribute('defer', element.getAttribute('defer'));
             }
-            if (element.async) {
-                newScript.async = element.async;
+            if (element.hasAttribute('async')) {
+                newScript.setAttribute('async',element.getAttribute('async'));
+            }
+            if (element.hasAttribute('onload')) {
+                newScript.setAttribute('onload', element.getAttribute('onload'));
             }
 
-            newScript.type = "text/javascript";
+            newScript.setAttribute('type', 'text/javascript');
```

Comment:
fyi here we might actually also want to check `getAttribute('type')`?

Reply:
Yes we should do that added same